sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs

```diff
@@ -2601,7 +2601,7 @@ public void UnexpectedException(string errorMessage)
         {
             if (IsEnabled())
             {
-                WriteEvent(125, partitionId ?? string.Empty, identifier ?? string.Empty, eventHubName ?? string.Empty, consumerGroup ?? string.Empty, errorMessage ?? string.Empty);
+                WriteEvent(125, partitionId ?? string.Empty, identifier ?? string.Empty, eventHubName ?? string.Empty, consumerGroup ?? string.Empty, operationId ?? string.Empty, errorMessage ?? string.Empty);
```

Comment:
Do we have a test that validates that operationId makes it to the event source message?

Reply:
We do not currently have any live tests that sniff the logs as they're written.   The validation that we do is over mocks, where the parameter was correctly being passed in.   We had considered adding a set, but I don't know that the ROI is there to justify it at this point.